plugins/BEdita/Core/src/Shell/Task/SetupAdminUserTask.php

```diff
@@ -113,6 +113,7 @@ public function main()
         $user->username = $this->param('admin-username');
         $user->password = $this->param('admin-password');
         $user->blocked = false;
+        $user->status = 'on';
```

Comment:
This is normally not needed because the `admin` user with ID 1 is created `locked` and with `status` on by the migrations. 

If for some reason `status` is not `on` we should first set `locked` to `false`, then save `status` `on` and then set `locked` to `true` again.

Reply:
I see. I think this was caused by a manually corrupted state, so we can ignore this change. I'll close the PR. Thanks!